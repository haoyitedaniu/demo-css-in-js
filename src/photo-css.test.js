import React from 'react'
import Photo from './photo-css'
import { render } from 'react-testing-library'

jest.mock('./global.css', () => ({
  customMedia: {
    '--photo-breakpoint': '(min-width: 30rem)',
  },
}))

it('renders successfully', () => {
  const { asFragment } = render(
    <Photo publicId="kitten" alt="A kitten" rounded />,
  )
  expect(asFragment()).toMatchSnapshot()
})
