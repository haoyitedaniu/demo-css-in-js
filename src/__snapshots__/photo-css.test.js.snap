// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders successfully 1`] = `
<DocumentFragment>
  <figure>
    <img
      class="roundedPhoto"
      sizes="(min-width: 30rem) 400px, 200px"
      src="https://res.cloudinary.com/demo/image/upload/c_scale,w_200/kitten"
      srcset="https://res.cloudinary.com/demo/image/upload/c_scale,w_200/kitten 200w, https://res.cloudinary.com/demo/image/upload/c_scale,w_400/kitten 400w, https://res.cloudinary.com/demo/image/upload/c_scale,w_800/kitten 800w"
    />
    <figcaption
      class="caption"
    >
      A kitten
    </figcaption>
  </figure>
</DocumentFragment>
`;
